doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -535,45 +535,66 @@ See the following instructions.
 
 .. _ug_nrf53_developing_ble_fota:
 
-.. include:: ../nrf52/developing.rst
+FOTA updates
+************
+
+.. include:: /device_guides/nrf52/fota_update.rst
    :start-after: fota_upgrades_intro_start
    :end-before: fota_upgrades_intro_end
 
-.. include:: ../nrf52/developing.rst
+FOTA over Bluetooth Low Energy
+==============================
```

Comment:
Please note the header level issue with the "FOTA update sample" section will be fixed in PR https://github.com/nrfconnect/sdk-nrf/pull/15708.